.circleci/config.yml

```diff
@@ -322,12 +322,26 @@ commands:
           dir_names: "ui e2e .circleci"
 
   puppeteer-access-test-user-setup:
+    parameters:
+      env_name:
+        description: The target environment for run Puppeteer end-to-end test. Must be one of "test", "staging", "local", "perf".
+        default: "test"
+        type: enum
+        enum: [ "test", "staging", "local", "perf" ]
     steps:
       - gcloud-auth-login
+      - run:
+          name: "Set environment variable ACCESS_TEST_USER"
+          working_directory: ~/workbench/e2e
+          command: |
+            cp .env.circleci.<< parameters.env_name >> .env
```

Comment:
nit: this took me a while to understand, because `<<`, `>>` are normal shell operators. Maybe put quotes around ".env.circleci.<<...>>" to make it slightly more obvious that this is one argument

Reply:
yeah, double quotes around it works. removed `source .env` as well.